examples/import_2.sh

```
@@ -0,0 +1,2 @@
+% export RHCS_CLIENT_ID="<your-client-id>"
```

Shellcheck (0.11.0)

[warning] This is interpreted as a command name ending with '%'. Double check syntax.

(SC2288)